pkg/adaptertesting/BUILD

```diff
@@ -4,6 +4,15 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
 go_library(
     name = "go_default_library",
-    srcs = ["builders.go"],
-    deps = ["//pkg/adapter:go_default_library"],
+    srcs = [
+        "adapters.go",
+        "builders.go",
+    ],
+    deps = [
+        "//pkg/adapter:go_default_library",
+        "//pkg/aspect:go_default_library",
+        "//pkg/aspect/denyChecker:go_default_library",
+        "//pkg/aspect/listChecker:go_default_library",
+        "//pkg/registry:go_default_library",
+    ],
```

Comment:
can we add a 'testonly' attribute to this library while we are here? i don't think we want this code used outside of tests.